java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/explorer/DefaultDecorationsImpl.java

```diff
@@ -56,6 +56,7 @@
     public static final String EXPLORER_ROOT = "Explorers"; // NOI18N
     public static final String COOKIES_EXT = "contextValues"; // NOI18N
     
+    public static final String CTXVALUE_FILE = "is:file"; // NOI18N
```

Comment:
Probably it should be documented somewhere.

Reply:
I think it is, in [`arch.xml`](https://github.com/apache/netbeans/pull/3414/files#diff-3d2e53a800fa8bf1c32071f2a26d981c1d4f698a9e85504fac40efaf3b402b25R964)